azure/notebooks/remote-explanation/azure-gpu-shap.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<p>If they download the config file from the portal, these steps are not needed. In this case, they're creating the workspace that hasn't been setup through portal. So we can either link the README to follow similar steps as other demos and use the portal or just comment it out and mention either way works</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
You can also load an existing workspace that's what the `try` block tries to do. It skips the need to download the `config.json` file and place it in the folder. I'll link it to the README and mention these are two different ways.